examples/server/utils.hpp

```diff
@@ -441,7 +441,7 @@ struct llama_server_response {
             {
                 LOG_VERBOSE("queue_results.push_back", {});
                 queue_results.push_back(result);
-                condition_results.notify_one();
+                condition_results.notify_all();
```

Comment:
@ggerganov @ngxson I was looking for a while why health/metrics/slots endpoints were blocking when a completion request is processing: only one caller thread was notified. Please confirm my fix.

Reply:
Thanks for noticing, seems like this is the root cause of many stability issues of concurrent requests.

Reply:
Yes, `notify_all` is correct